.github/workflows/ci.yml

```diff
@@ -1,19 +1,27 @@
 name: ci
+
 on:
   push:
   pull_request:
+
 jobs:
   build:
-    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        os: [ubuntu-latest, macos-latest]
```

Comment:
What's the advantage of running CI on macOS as well?

Reply:
My thinking was since that's the platform on which many engineers work, it's a good idea to make sure we don't introduce any incompatibilities.